README.rst

```diff
@@ -37,16 +37,49 @@ By default, SCORM modules will be accessible at "/scorm/" urls and static assets
 
     XBLOCK_SETTINGS["ScormXBlock"] = {
         "LOCATION": "alternatevalue",
+        "SCORM_FILE_STORAGE_TYPE": "openedx.features.clearesult_features.backend_storage.ScormXblockS3Storage"
     }
 
+If you are using this xblock locally, there a configuration variable whose value you will have to set empty string::
+
+    XBLOCK_SETTINGS["ScormXBlock"] = {
+        "SCORM_MEDIA_BASE_URL": ""
+    }
+
+You can face x-frame-options restrictions. For that, you can use any workaround. Like for local you can install any extension on chrome which will disable x-frame-restrictions.
+For example: ``Ignore X-Frame headers`` Google Chrome extension.
```

Comment:
Add link to the extension if we are adding this in the instructions?

Reply:
done.